espnet/nets/pytorch_backend/e2e_tts_tacotron2.py

```diff
@@ -283,94 +286,108 @@ class Tacotron2(TTSInterface, torch.nn.Module):
-                            help='Zoneout rate')
-        parser.add_argument('--reduction-factor', default=1, type=int,
-                            help='Reduction factor')
+        group.add_argument('--use-batch-norm', default=True, type=strtobool,
+                           help='Whether to use batch normalization')
+        group.add_argument('--use-concate', default=True, type=strtobool,
+                           help='Whether to concatenate encoder embedding with decoder outputs')
+        group.add_argument('--use-residual', default=True, type=strtobool,
+                           help='Whether to use residual connection in conv layer')
+        group.add_argument('--dropout-rate', default=0.5, type=float,
+                           help='Dropout rate')
+        group.add_argument('--zoneout-rate', default=0.1, type=float,
+                           help='Zoneout rate')
+        group.add_argument('--reduction-factor', default=1, type=int,
+                           help='Reduction factor')
+        group.add_argument("--spk-embed-dim", default=None, type=int,
+                           help="Number of speaker embedding dimensions")
+        group.add_argument("--spc-dim", default=None, type=int,
+                           help="Number of spectrogram dimensions")
         # loss related
-        parser.add_argument('--use-masking', default=False, type=strtobool,
-                            help='Whether to use masking in calculation of loss')
-        parser.add_argument('--bce-pos-weight', default=20.0, type=float,
-                            help='Positive sample weight in BCE calculation (only for use-masking=True)')
-        parser.add_argument("--use-guided-attn-loss", default=False, type=strtobool,
-                            help="Whether to use guided attention loss")
-        parser.add_argument("--guided-attn-loss-sigma", default=0.4, type=float,
-                            help="Sigma in guided attention loss")
-        return
-
-    def __init__(self, idim, odim, args):
+        group.add_argument('--use-masking', default=False, type=strtobool,
+                           help='Whether to use masking in calculation of loss')
+        group.add_argument('--bce-pos-weight', default=20.0, type=float,
+                           help='Positive sample weight in BCE calculation (only for use-masking=True)')
+        group.add_argument("--use-guided-attn-loss", default=False, type=strtobool,
+                           help="Whether to use guided attention loss")
+        group.add_argument("--guided-attn-loss-sigma", default=0.4, type=float,
+                           help="Sigma in guided attention loss")
+        return parser
+
+    def __init__(self, idim, odim, args=None):
         # initialize base classes
         TTSInterface.__init__(self)
         torch.nn.Module.__init__(self)
 
+        # get default arguments and fill missing arguments
```

Comment:
I think we should split this to `espnet.net.args.py` or somewhere we can reuse this for the other models